lib/internal/http2/core.js

```diff
@@ -2313,6 +2313,9 @@ function initializeOptions(options) {
   options.allowHalfOpen = true;
   assertIsObject(options.settings, 'options.settings');
   options.settings = Object.assign(Object.create(null), options.settings);
+  options.Http2ServerRequest = options.Http2ServerRequest || Http2ServerRequest;
+  options.Http2ServerResponse = options.Http2ServerResponse ||
+    Http2ServerResponse;
```

Comment:
Could you line up Http2ServerResponse with the left side of `options.Http2ServerResponse` or bring both down to the next line? Thanks!